apps/modernization-ui/src/apps/page-builder/components/QuickConditionLookup/QuickConditionLookup.tsx

```diff
@@ -58,61 +52,57 @@ export const QuickConditionLookup = ({ modal, addConditions }: Props) => {
         addConditions(selectedConditions);
     };
 
-    const handleSubmitSearch = useCallback(
-        (page: number) => {
-            setLoading(true);
-            const authorization = `Bearer ${state.getToken()}`;
-            const search: ReadConditionRequest = { searchText };
+    const handleSubmitSearch = useCallback(() => {
+        setLoading(true);
+        const authorization = `Bearer ${state.getToken()}`;
+        const search: ReadConditionRequest = { searchText };
 
-            ConditionControllerService.searchConditionsUsingPost({
-                authorization,
-                search,
-                page: page,
-                size: pageSize
+        searchConditions(authorization, currentPage, pageSize, `${sortBy},${sortDirection}`, {
+            searchText: search.searchText
+        })
```

Comment:
Issue: the `currentPage` starts at 1 and corresponds to what the user sees. The API expects pages to start at 0. I think we need a `currentPage - 1` when performing a search